spec/spec_helper.rb

```diff
@@ -99,7 +99,7 @@
   #   Kernel.srand config.seed
 
   # Mock all requests to symphony for feature tests
-  config.before type: :feature do
+  config.before :example, type: Proc.new { :service || :feature } do
```

Comment:
`config.before type: :feature do` not working? I did not know this 🤔

Reply:
It is working, I just wanted to add the sinatra fake web service to service specs too so I didn't have to do a whole bunch of redundant mocking

Reply:
ahhh yes cool! I just did not realize the service part :facepalm: